### ISSUE TITLE
Add leetcode.cn

### ISSUE THREAD

**Issue by worker2011** [aropan/clist#341]
As the global website of Leetcode, [https://leetcode.com](https://leetcode.com) has been added to Clist, but Chinese users are desiring to add [https://leetcode.cn/](https://leetcode.cn/), the Chinese Edition of Leetcode, which all Chinese users are recommended to do problems here.

**Comment by aropan** [comment]
I don't want add separate resource leetcode.cn, because it will duplicate information from leetcode.com. Chinese users are tracked in global leetcode.com. If something is not convenient, then let me know.

**Comment by worker2011** [comment]
> I don't want add separate resource leetcode.cn, because it will duplicate information from leetcode.com. Chinese users are tracked in global leetcode.com. If something is not convenient, then let me know.

1. Chinese website and the international website are different, that means the accounts are different, that is, if you have an account in the Chinese site, you cannot log in to the global site as they are different accounts.
2. [The rank](https://leetcode.cn/contest/) of Chinese users have two for the contests: one is in the Chinese site, another is global site. That is absolutely important.

To sum up, you do not have to worry about duplicating the data of Chinese users.

**Comment by aropan** [comment]
Current leetcode.com resource (https://clist.by/resource/leetcode.com/) have all accounts (for international and Chinese websites).

Standings and problems for leetcode.cn will be duplicate leetcode.com. 

What is mean "don't worry about duplicating"? Chinese users will be duplicate these users for leetcode.com also.

**Comment by aropan** [comment]
All users for leetcode.com resource have suffix "@.com" or "@.cn" in handle to indicate original website for user.

**Comment by worker2011** [comment]
That is something I want. So now could I suggest to change the website option from "leetcode.com" to "leetcode.com & leetcode.cn" or so? After that, I believe there will not be anyone be confused and creat these issues like me.

**Comment by aropan** [comment]
By default leetcode.com have Chinese users ([rating](https://leetcode.com/contest/globalranking/) and [standings](https://leetcode.com/contest/weekly-contest-393/ranking)).
